modules/reindex/src/test/resources/rest-api-spec/test/delete_by_query/20_validation.yaml

```diff
@@ -6,6 +6,21 @@
         index: _all
 
 ---
+"no query logs deprecation":
+
+  - skip:
+      features: "warnings"
+      version: " - 5.2.99"
```

Comment:
Cool, I thought the version/reason had to do with introducing warnings in the YAML framework :-)